src/include/homestore/btree/detail/btree_node.hpp

```diff
@@ -527,15 +537,10 @@ class BtreeNode : public sisl::ObjLifeCounter< BtreeNode > {
 
     virtual uint32_t occupied_size() const { return (node_data_size() - available_size()); }
     bool is_merge_needed(const BtreeConfig& cfg) const {
-#if 0
 #ifdef _PRERELEASE
-       if (iomgr_flip::instance()->test_flip("btree_merge_node") && occupied_size() < node_data_size) {
-           return true;
-       }
-
-       auto ret = iomgr_flip::instance()->get_test_flip< uint64_t >("btree_merge_node_pct");
-       if (ret && occupied_size() < (ret.get() * node_data_size() / 100)) { return true; }
-#endif
+        if (min_keys_in_node()) {
```

Comment:
what if we need min_keys_in_node is 0? should we set it as 1?

Reply:
Since here's returning `total_entries() < min_keys_in_node()`, I don't think setting it to 0 can be meaningful. Or do you mean using `min_keys_in_node=0` to disable merge?